src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -313,69 +356,107 @@ kept, or `subsampled50` if only 50% of the streamlines were kept.
 ### File formats
 
 -   [.trk - Diffusion Toolkit (+TrackVis)](https://web.archive.org/web/20190103230122/http://www.trackvis.org/docs/?subsect=fileformat)
--   [.tck - MRTrix](https://web.archive.org/web/20190103230209/https://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html)
+-   [.tck - MRtrix](https://web.archive.org/web/20190103230209/https://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html)

```

Comment:
Could you add the anchor to go to the `.tck` format directly in the link? It's `#tracks-file-format-tck`

Reply:
Resolved by 74ff6d0f.